homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -35,6 +51,41 @@
 
 SUCCESS = ["ok"]
 
+SensorType = namedtuple("SensorType", ["unit", "icon", "device_class"])
```

Comment:
We can use a dataclasses class instead.
https://docs.python.org/3/library/dataclasses.html

Reply:
done